.github/workflows/ci.yml

```diff
@@ -73,12 +73,27 @@ jobs:
         with:
           python-version: ${{matrix.python-version}}
           architecture: "x64"
-      - run: sudo apt update && sudo apt install -y libhdf5-dev
+      - name: Install HDF5
+        if: matrix.part == 3
+        run: sudo apt update && sudo apt install -y libhdf5-dev
       - run: pip install -r requirements.txt
-      - run: python -m pytest --testmon-noselect modin/pandas/test/test_io.py
+      - run: bash run-tests.sh ${{matrix.engine}} -k "TestDataFrame${{matrix.part}}"
+        if: matrix.part != 3
+      - run: python -m pytest modin/pandas/test/test_series.py
+        if: matrix.part == 3
+      - run: python -m pytest modin/pandas/test/test_concat.py
+        if: matrix.part == 3
+      - run: python -m pytest modin/pandas/test/test_groupby.py
+        if: matrix.part == 3
+      - run: python -m pytest modin/pandas/test/test_reshape.py
+        if: matrix.part == 3
+      - run: python -m pytest modin/pandas/test/test_general.py
+        if: matrix.part == 3
+      - run: python -m pytest modin/pandas/test/test_io.py
         if: matrix.part == 3
-      - run: python -m pytest --testmon-noselect modin/experimental/pandas/test/test_io_exp.py
```

Comment:
Why `--testmon-noselect` was be removed?

Reply:
We used to use testmon Test Monitor to only run tests that had changed, but it was not stable enough to be used when there were many changes. This flag doesn't do anything anymore.